### DISCUSSION TITLE
Motor para crear un slider solo con CSS

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#30] [Announcements]
# Motor para crear un slider solo con CSS

Un slider creado con CSS puro sin javascript. Diseñado para ser lo más ligero y rápido posible.

https://www.zkreations.com/2014/08/slider-css3-responsive.html

**Comment by TakoMics** [comment]
Antes que nada, Feliz Navidad, Daniel, espero la hayas pasado excelente.

Este aporte, como lo dije en su tiempo, hace años, cuando salió por primera vez este slider, está increíble. Un slider siempre le da dinamismo y actualización a una web, ya que la herramienta se presta mucho para añadir novedades, anuncios y destacar contenido.

Me alegra muchísimo que poco a poco Zkreations se esté llenando otra vez de material y, sobre todo, de aquellos aportes míticos y de antaño que han hecho a ZKreations único.

Saludos y felices fiestas. Un abrazo desde acá.

**Reply by danieIabel** [reply]
Buenos días amigo, muchas gracias por tu comentario y felicitaciones. Yo también te deseo unas felices fiestas desde aquí, y te invito a que esperes todo lo nuevo que tengo para 2024, estoy seguro de que te encantará lo que está por venir. ¡Un abrazo!
